### PR TITLE
Clean up

### LINKED ISSUE #16: Typo: builded
I think "builded" exists but is archaic. This need to be replaced to "built".

### LINKED ISSUE #18: Consider using the word `run` instead of `execute`
It's shorter... is it better?

### LINKED ISSUE #19: Add logs in watch loop
The command executed might not be providing much input. It's important to add info logs that tells the user when the command starts and when it stops.

### LINKED ISSUE #21: Re-export cargo_metadata
So the user can get Version

### PR DESCRIPTION
**Pull request by yozhgoor**
This PR will close some tickets, clean up the code when needed and add some missing stuffs.

First commit closes #16 
Second commit closes #18 
Third commit closes #21
Fifth commit closes #19

### CODE REVIEW

**Comment by yozhgoor** [comment]
close #16

File: src/lib.rs [hunk 1]
```diff
@@ -282,7 +284,22 @@ impl Watch {
...
-    pub fn execute(mut self, mut command: process::Command) -> Result<()> {
+    fn is_excluded_path(&self, path: &Path) -> bool {
+        self.exclude_paths.iter().any(|x| path.starts_with(x))
+            || self
+                .workspace_exclude_paths
+                .iter()
+                .any(|x| path.starts_with(x))
+    }
+
+    fn is_hidden_path(&self, path: &Path) -> bool {
```
Comment (cecton) [inline src/lib.rs:295]: You don't need self here

**Review by cecton — APPROVED** [review]

File: src/lib.rs [hunk 2]
```diff
@@ -344,28 +363,14 @@ impl Watch {
                         }
                     }
 
+                    log::trace!("Relaunching given command");
```
Comment (cecton) [inline src/lib.rs:366]:
```suggestion
                    log::info!("Changes detected. Re-running command.");
```

**Review by cecton — COMMENTED** [review]
I said info log ^_^ This one is for the end user

**Review by cecton — APPROVED** [review]

**Pull request merged**